pkg/operator/config/config_test.go

```diff
@@ -258,4 +260,78 @@ func TestBuildConfigLogs(t *testing.T) {
 	}
 }
 
-func strPointer(s string) *string { return &s }
+func TestBuildConfigIntegrations(t *testing.T) {
+	in := `
+  Agent:
```

Comment:
I don't really know what to do about the weird indentation here. I can't put tabs in the yaml without wrapping it in a call to util.Untab.

Reply:
I changed it to tabs 🤷